src/shared/containers/index.tsx

```diff
@@ -43,6 +43,12 @@ export const ClientLibrariesPage = lazy(() =>
 export const TelegrafPluginsPage = lazy(() =>
   import('src/writeData/containers/TelegrafPluginsPage')
 )
+export const EditSecretOverlay = lazy(() =>
+  import('src/secrets/components/EditSecretOverlay')
+)
+export const CreateSecretOverlay = lazy(() =>
+  import('src/secrets/components/CreateSecretOverlay')
+)
```

Comment:
I thought I had deleted this. They're not being used

Reply:
Removed.